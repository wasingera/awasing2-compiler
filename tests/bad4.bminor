what if we went exploring
\just \kidding
function: void explore() {
""""""
}
