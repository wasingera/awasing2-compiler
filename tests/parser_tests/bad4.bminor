print "big big chungus big chungus big chungus";

x: string = "end of the line soldier";
