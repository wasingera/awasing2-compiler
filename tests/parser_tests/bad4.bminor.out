parse error: syntax error
Parse failed: print
